### ISSUE TITLE
The evaluation strategy (PCKh)

### ISSUE THREAD

**Issue by tom1994** [xingyizhou/pytorch-pose-hg-3d#18]
Thanks for the exciting code!
But, I'm a little confused with the evaluation strategy of PCKh.
Here is the code (at 'src/utils/eval.py: line 33'):
```python
# PCKh0.5
def Accuracy(output, target):
    preds = getPreds(output)
    gt = getPreds(target)
    # ref.outputRes = 64
    dists = calcDists(preds, gt, np.ones(preds.shape[0]) * ref.outputRes / 10)
    acc = np.zeros(len(ref.accIdxs))
    avgAcc = 0
    badIdxCount = 0

    for i in range(len(ref.accIdxs)):
        acc[i] = distAccuracy(dists[ref.accIdxs[i]])
        if acc[i] >= 0:
            avgAcc = avgAcc + acc[i]
        else:
            badIdxCount = badIdxCount + 1

    if badIdxCount == len(ref.accIdxs):
        return 0
    else:
        return avgAcc / (len(ref.accIdxs) - badIdxCount)
```
the `calcDists` function calculates the normalized distances of each predicted joint with ground truth. if we used PCKh [*M. Andriluka, L. Pishchulin, P. Gehler, and B. Schiele. 2D human pose estimation: New benchmark and state of the art analysis. In CVPR, 2014. 6, 7*] as the evaluation strategy，the normalize should be the head size.
But I noticed that the normalize is a matrix with the fixed value (=6.4) not the size of the head.
So, why use the fixed value to replace the head size? 
Thanks 😃

**Comment by xingyizhou** [comment]
Hi, 
  This is not the final evaluation code, but just a quit approximation during training. For the formal evaluation defined on the MPII paper, please run `src/tools/eval_PCKh.py` at the `2D` branch. It will give a higher result than what you see during training.

**Comment by tom1994** [comment]
@xingyizhou Got it, thank you very much 😃
